pages/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+import Image from 'next/image';
+import Link from 'next/link';
+
+const Navbar = () => {
+  const GITHUB_LOGO = "/icons/github-white.svg";

```

Comment:
please use proper import statements to import the images

Reply:
I didn't understand this vaibhav?